### PR TITLE
Make it build with ghc 9.10

### PR DESCRIPTION
**Pull request by erikd**

### CODE REVIEW

File: cabal.project [hunk 1]
```diff
@@ -38,3 +38,18 @@ import: ./asserts.cabal
...
+  -- All these cardano-ledger packages have been fixed on master but not
+  -- yet released to CHaP.
+  , cardano-ledger-alonzo:plutus-ledger-api
+  , cardano-ledger-alonzo-test:plutus-ledger-api
+  , cardano-ledger-babbage:plutus-ledger-api
+  , cardano-ledger-binary:plutus-ledger-api
+  , cardano-ledger-conway:plutus-ledger-api
+  -- Upstream maintainer seems reluctant to fix this.
+  -- https://github.com/phadej/latex-svg/issues/10
+  , latex-svg-image:base
```
Comment (jasagredo) [inline cabal.project:52]: Notice we are using our own version from CHaP, not the one on Hackage

File: cabal.project [hunk 2]
```diff
@@ -38,3 +38,18 @@ import: ./asserts.cabal
 if(os(windows))
   constraints:
     bitvec -simd
+
+allow-newer:
```
Comment (jasagredo) [inline cabal.project:42]: Couldn't we wrap this in a conditional for the GHC version?

Comment (erikd) [reply cabal.project:42]: I hope to remove these before  pull it out of draft mode.

File: cabal.project (hunk 1 above)
Comment (erikd) [reply cabal.project:52]: Thanks, I will try updating the one in CHaP.

Comment (jasagredo) [reply cabal.project:52]: It is a fork of mine (I opened a PR/issue against the upstream one long ago and it is still waiting there). Let me give you permissions for pushing

Comment (erikd) [reply cabal.project:52]: Add a revision for the one in CHaP. PR incoming for https://github.com/jasagredo/latex-svg

Comment (erikd) [reply cabal.project]: PR submitted.

Now using the fixed version from CHaP.

File: cabal.project (hunk 2 above)
Comment (erikd) [reply cabal.project]: It is some time before the `ledger` packages are released to CHaP so I have made them conditional on GHC version.

**Pull request merged**